README.md

```diff
@@ -1,66 +1,77 @@
+The ArcGIS Runtime SDK for .NET Toolkit contains controls and components you can use to accelerate your development with [ArcGIS Runtime SDK for .NET](https://developers.arcgis.com/net/).
 
-The ArcGIS Runtime SDK for .NET Toolkit contains controls and utilities you can use with [ArcGIS Runtime SDK for .NET](http://links.esri.com/dotnetsdk).
+[![Link: Toolkit Guide](https://img.shields.io/badge/%F0%9F%93%84-Toolkit%20Guide-633b9b?style=flat-square)](https://esri.github.io/arcgis-toolkit-dotnet/controls.html)
+[![Link: Toolkit API Reference (landing page)](https://img.shields.io/badge/Toolkit%20API%20Reference:-fff?style=flat-square)](https://esri.github.io/arcgis-toolkit-dotnet/api/index.html)
+[![Link: Toolkit API Reference - Android](https://img.shields.io/badge/Xamarin.Android-3ddc84?style=flat-square&labelColor=gray&logo=android)](https://esri.github.io/arcgis-toolkit-dotnet/api/android/index.html)
+[![Link: Toolkit API Reference - iOS](https://img.shields.io/badge/Xamarin.iOS-black?style=flat-square&labelColor=gray&logo=ios)](https://esri.github.io/arcgis-toolkit-dotnet/api/ios/index.html)
+[![Link: Toolkit API Reference - Forms](https://img.shields.io/badge/Xamarin.Forms-3498db?style=flat-square&labelColor=gray&logo=Xamarin)](https://esri.github.io/arcgis-toolkit-dotnet/api/netstd/index.html)
+[![Link: Toolkit API Reference - WPF](https://img.shields.io/badge/WPF-0078d6?style=flat-square&labelColor=gray&logo=windowsxp)](https://esri.github.io/arcgis-toolkit-dotnet/api/netfx/index.html)
+[![Link: Toolkit API Reference - UWP](https://img.shields.io/badge/UWP-0078d6?style=flat-square&labelColor=gray&logo=windows)](https://esri.github.io/arcgis-toolkit-dotnet/api/uwp/index.html)
 
-There are two ways to add Toolkit to your project:
+| <img width="150" alt="Basemap Gallery" src="https://user-images.githubusercontent.com/29742178/124198151-f2dc6380-da84-11eb-8e78-4e705d14c33d.png" /> | <img alt="Search View" width="150" src="https://user-images.githubusercontent.com/29742178/142301018-4bbeb0f2-3021-49a7-b5ec-f642c5700bd0.png" /> | <img src="https://user-images.githubusercontent.com/29742178/121975740-34f07000-cd37-11eb-9162-462925cb3fe7.png" width="150" alt="Overview Map" /> | <img src="https://user-images.githubusercontent.com/1378165/73389924-011fc200-428a-11ea-91bf-4ea1c2bf6683.png" width="105" alt="Legend" /> |
```

Comment:
A `title` attribute would be good to show the component name when mousing over an image (same text as `alt`?).
```suggestion
| <img width="150" alt="Basemap Gallery" title="Basemap Gallery" src="https://user-images.githubusercontent.com/29742178/124198151-f2dc6380-da84-11eb-8e78-4e705d14c33d.png" /> | <img alt="Search View" title="Search View" width="150" src="https://user-images.githubusercontent.com/29742178/142301018-4bbeb0f2-3021-49a7-b5ec-f642c5700bd0.png" /> | <img src="https://user-images.githubusercontent.com/29742178/121975740-34f07000-cd37-11eb-9162-462925cb3fe7.png" width="150" alt="Overview Map" title="Overview Map"/> | <img src="https://user-images.githubusercontent.com/1378165/73389924-011fc200-428a-11ea-91bf-4ea1c2bf6683.png" width="105" alt="Legend" title="Legend"/> |
```